_posts/2024-05-04-authorship-info.markdown

```diff
@@ -0,0 +1,60 @@
+---
+#### Blog Post Template ####
+
+#### Post Information ####
+title: "Note on Inline Authorship Information in scikit-learn"
+date: May 4, 2024
+
+#### Post Category and Tags ####
+# Format in titlecase without dashes (Ex. "Open Source" instead of "open-source")
+categories:
+  - Updates
+tags:
+  - Open Source
+  - Machine Learning
+  - License
+
+#### Featured Image ####
+featured-image: BSD_watermark.svg
+
+#### Author Info ####
+# Can accomodate multiple authors
+# Add SQUARE Author Image to /assets/images/author_images/ folder
+postauthors:
+  - name: Adrin Jalali
+    website: https://adrin.info/
+    image: adrin-jalali.jpeg
+---
+<div>
+  <img src="/assets/images/posts_images/{{ page.featured-image }}" alt="">
+  {% include postauthor.html %}
+</div>
+
+Historically, scikit-learn's files have included authorship information similar
+to the following format:
+
+```python
+# Authors: Author1, Author2, ...
+# License: BSD 3 clause
+```
+
+However, after a series of discussions which you can see in detail in [this
+issue]( https://github.com/scikit-learn/scikit-learn/pull/28799), could list
+the following caveats to the status quo:
+
+- Authorship information was not up-to-date and in most cases, but not always,
+  reflect the original authors of the file;
```

Comment:
```suggestion
  does not reflect the original authors of the file;
```
Did you forget a "not"?

Reply:
no I didn't, I think in most cases people do NOT update the authorship, so in most cases it does reflect the original authors.